apps/artist/routes.coffee

```diff
@@ -18,6 +18,7 @@ sd = require('sharify').data
 
 @index = (req, res, next) ->
   tab = if req.params.tab? then req.params.tab else ''
+  showArtistCTA = req.query.show_artist_cta_code is SHOW_ARTIST_CTA_CODE
```

Comment:
In the config file `SHOW_ARTIST_CTA_CODE` is assigned as null. I'm a little confused as to how this gets set here.